src/utils/genresOptions.ts

```diff
@@ -0,0 +1,12 @@
+import { Genre } from '../types/Genre';
+
+export const genresOptions = (genres: Genre[]) => {
+  const options = genres.map(genre => {
+    const option: any = {};
+    option.value = genre.id;
+    option.label = genre.name;
+    return option;
+  });
+
+  return options;
+};
```

Comment:
this function could be optimized and it would be better not to use `any` type
```suggestion

type Option = { value: number, label: string };

export const genresOptions = (genres: Genre[]): Option[] => genres.map(genre => ({
  value: genre.id, label: genre.name,
}));
```